platform/linux-generic/Makefile.am

```diff
@@ -276,10 +276,15 @@ if WITH_ARMV8_CRYPTO
 __LIB__libodp_linux_la_SOURCES += \
 			   arch/aarch64/odp_crypto_armv8.c
 else
+if WITH_IPSECMB_CRYPTO
+__LIB__libodp_linux_la_SOURCES += \
+			   odp_crypto_ipsecmb.c
```

Comment:
Generally, each individual commit merged to the ODP repo should build and pass tests successfully. So, this commit should be squashed with the second commit for `odp_crypto_ipsecmb.c` to be available.

Reply:
Make sense, first two comments squashed.